### PR TITLE
run build test with Both buildkit and classic builder

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Run build tests twice, once with builder classic and then with buildkit enabled

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: pkg/e2e/compose_build_test.go [hunk 1]
```diff
@@ -26,7 +27,18 @@ import (
 	"gotest.tools/v3/icmd"
 )
 
-func TestLocalComposeBuild(t *testing.T) {
+func TestLocalComposeBuild_Classic(t *testing.T) {
+	os.Setenv("DOCKER_BUILDKIT", "0")    //nolint:errcheck
```
Comment (mat007) [inline pkg/e2e/compose_build_test.go:31]: nit: here we could either do
```suggestion
	_ = os.Setenv("DOCKER_BUILDKIT", "0")
```
or check the error?

**Review by mat007 — COMMENTED** [review]
Test is failing, it looks like a real error?
(@ulyssessouza ?)

File: pkg/e2e/compose_build_test.go (hunk 1 above)
Comment (ndeloof) [reply pkg/e2e/compose_build_test.go:31]: sorry, was not intended to be merged, just wanted to do a quick and dirty try to get both builder classic and buildkit CI run

**Comment by mat007** [comment]
@ndeloof should we close this PR?

**Pull request closed without merging**